test/test_attention.py

```diff
@@ -43,9 +43,9 @@ def generate_broadcast_mask(
         torch.randint(1, seq_length, (batch,), device="cuda")[:, None]
         > torch.arange(0, seq_length, device="cuda")[None, :]
     )
-    attention_mask = attention_mask.to(dtype)
     attention_mask = torch.reshape(attention_mask, (batch, 1, 1, seq_length))
-    attention_mask = (1.0 - attention_mask) * torch.finfo(dtype).min
+    attention_mask = torch.where(attention_mask, 0, float("-inf"))
```

Comment:
not sure to understand this change

Reply:
I just wanted to use real infinite value and no more min value. On original attention triton kernel, minus infinite doesn't NAN